builder.go

```diff
@@ -94,10 +95,30 @@ func buildAndRunDir(dir string, filters []string, goBuildTags string) error {
 	return nil
 }
 
+func goMod() string {
+	if _, err := os.Stat("go.mod"); os.IsNotExist(err) {
+		return ""
+	}
+	file, err := os.Open("go.mod")
+	defer file.Close()
+	if err != nil {
+		panic(err)
```

Comment:
panicing here isn't consistent with `return ""` above. If we're ok with skipping the mod file when unavailable, maybe log this to stderr but don't panic.